### PR TITLE
Update DevTest Labs existing schema to match recently published swagger defn

### PR DESCRIPTION
**Pull request by ryanvog**
API spec PR: https://github.com/Azure/azure-rest-api-specs/pull/1067

### CODE REVIEW

**Comment by TianoMS** [comment]
@ryanvog I believe the updated file is a swagger file, which is not a resource schema file.

**Comment by ryanvog** [comment]
@TianoMS Thanks! I pasted the wrong source.  I just updated the PR with the correct source.

File: schemas/2016-05-15/Microsoft.DevTestLab.json [hunk 1]
```diff
@@ -114,14 +136,29 @@
...
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties"
+      ],
+      "description": "Microsoft.DevTestLab/labs/artifactsources"
+    },
+    "labs_costs": {
```
Comment (TianoMS) [inline schemas/2016-05-15/Microsoft.DevTestLab.json]: could you update tests/2016-05-15/Microsoft.DevTestLab.tests.json for your changes?

Comment (ryanvog) [inline schemas/2016-05-15/Microsoft.DevTestLab.json]: Tests added

**Pull request merged**